index.html

```diff
@@ -0,0 +1,111 @@
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  <link href="style.css" rel="stylesheet" type="text/css">
+  <title>Homepage of YHS</title>
+</head>
+
+<body>
+
+  <header class="container">
+    <nav class="nav navbar-one">
+      <a class="nav-item" href="#"><i class="fa fa-facebook fa_custom"></i></a>
+      <a class="nav-item" href="#"><i class="fa fa-twitter"></i></a>
+      <a class="nav-item" href="#">Logout</a>
+      <a class="nav-item" href="index.html">Homepage</a>
+      <a class="nav-item" href="#">English</a>
+    </nav>
+    <nav class="navbar-two">
+      <div class="navbar-header">
+        <div class="humburger">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </div>
+        <a id="header-logo" href="index.html"><img src="./img/NewLogo4.png" alt="Logo"></a>
+      </div>
+      <div class="collapse navbar-collapse" id="myNavbar">
+        <a class="link" href="index.html">Home</a>
+        <a class="link" href="about.html">About</a>
+        <a class="link" href="#">Program</a>
+        <a class="link" href="#">Join</a>
+        <a class="link" href="#">Sponsors</a>
+        <a href="#" id="campaign" class="link">YHS Campaign</a>
+      </div>
+    </nav>
+  </header>
+
+  <!--#################### MAIN SECTION ###################-->
+  <main class="main-container">
+    <section class="landing-page-section">
+      <p class="greetings">"Hello! Welcome to YHS"</p>
+      <h1 class="landing-page-header">YOUTH HOPE SUMMIT <br>2022</h1>
+      <p class="landing-page-para">
+        A joyful celebration believing in the value of opennes and sharing, creating a positive change with people from
+        over 80 countries is taking place in August, in Ghana.
+      </p>
+      <h2 class="duration">2022.08.19 (FRI) ~ 20(SAT)</h2>
+      <p class="location">@ National Theater, Accra-Ghana</p>
+    </section>
+
+    <section class="activities">
+      <h2 class="activities-header">Main Program</h2>
+      <div class="row">
+        <div class="col-row">
+          <img class="presentation" src="./img/lecture.png" alt="Lecture">
+          <h3 class="lecture-heading">Lecture</h3>
+          <p class="lecture-para">Listen to the speakers as they share their knowledge and experiences on the various
+            topics given them.</p>
+        </div>
+        <div class="col-row">
+          <img class="forum" src="./img/forum.png" alt="Forum">
+          <h3 class="forum-heading">Forum</h3>
+          <p class="forum-para">Have the time to share your throughts and opinions with expert for each topic</p>
+        </div>
+        <div class="col-row">
+          <img class="connect" src="./img/connect.png" alt="Connect">
+          <h3 class="ignite-heading"> YHS Connect</h3>
+          <p class="ignite-para">Get opportunities to network with attendees around the world also after the summit.
```

Comment:
- [ ] Kindly add 2 more cards as shown in the attached loom video as per the project requirements.

Reply:
@sja-thedude I honestly appreciate your review. It was detailed. I will be submitting the correction soon. However, with the more button, since it is optional, I would like to implement it after the review.
![Capture](https://user-images.githubusercontent.com/92553079/175347198-1d68f2ec-25be-42dc-8eae-e404341e19bc.PNG)
- Reference to the main page also, The project required that we personalise it so that is why I have to use my three cards because if I add more I will not know what to write. 
Thank you